fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentClientMethodMapper.java

```diff
@@ -31,22 +31,19 @@ protected JavaVisibility methodVisibility(ClientMethodType methodType, boolean h
                     break;
 
                 case SimpleAsync:
+                case SimpleSync:
                     visibility = NOT_GENERATE;
                     break;
 
                 default:
-                    visibility = super.methodVisibility(methodType, hasContextParameter, isProtocolMethod);
+                    visibility = super.methodVisibility(methodType, true, isProtocolMethod);
                     break;
             }
         } else {
-            switch (methodType) {
-                case PagingAsyncSinglePage:
-                    visibility = NOT_VISIBLE;
-                    break;
-
-                default:
-                    visibility = super.methodVisibility(methodType, hasContextParameter, isProtocolMethod);
-                    break;
+            if (methodType == ClientMethodType.PagingAsyncSinglePage || methodType == ClientMethodType.SimpleSyncRestResponse) {
```

Comment:
Good catch, I'll update this to prevent single page from generating for fluent gen. Diving further I think a better pattern moving forward would be having ClientMethodMapper just generate everything consistently and having each generation type having an override like this to prevent what shouldn't be generated.